libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -579,8 +582,8 @@ void SRV_Channels::set_trim_to_min_for(SRV_Channel::Aux_servo_function_t functio
 void SRV_Channels::set_default_function(uint8_t chan, SRV_Channel::Aux_servo_function_t function)
 {
     if (chan < NUM_SERVO_CHANNELS) {
-        int8_t old = channels[chan].function;
```

Comment:
Was this a bug?  function is an AP_Int16 (now AP_Enum16) and can't fit in 8 bits.